utils/validationRules/Replies/updateReplySchema.js

```diff
@@ -0,0 +1,27 @@
+const Joi = require("@hapi/joi");
+
+/**
+ * Schema validation for PATCH '/comments/{commentId}/replies/{replyId}'
+ */
+
+const updateReplySchema = {
+  options: {
+    allowUnkown: true,
```

Comment:
There is a typo here:
```suggestion
    allowUnknown: true,
```